rmw_opensplice_cpp/CMakeLists.txt

```diff
@@ -26,6 +26,8 @@ endif()
 
 find_package(rmw REQUIRED)
 find_package(rosidl_generator_cpp REQUIRED)
+find_package(std_msgs REQUIRED)
+find_package(introspection_msgs REQUIRED)
```

Comment:
I think this is going to be a problem with this approach. I do not believe it makes sense to have our rmw implementations depend on and use our messages.

Even though it might be more work, I think the right thing to do is to figure out how to expose the needed information up through the rmw interface and then any work that needs to publish or subscribe to topics should be done in `rcl`.

I'll have to see what all is happening under the hood in order to make a more concrete suggestion.